jobfunnel/config/settings.yaml

```diff
@@ -7,9 +7,11 @@ output_path: 'search'
 
 # providers from which to search (case insensitive)
 providers:
+  - 'GlassDoorStatic'
```

Comment:
also here, why did we have glassdoor disabled currently?

Reply:
Glassdoor isn't disabled here, it just changed to GlassDoorStatic since that's the provider being tested.